packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -134,6 +137,13 @@ Summary: Modules for the Linux kernel
 %description modules
 %{summary}.
 
+%package mkfs-xfs-conf
+Summary: mkfs configurations for the XFS filesystem
+Provides: %{name}(mkfs-xfs-conf)
```

Comment:
Same as elsewhere, this `Provides` is added here and removed in the last commit of the series